api/ssi-bridge/src/services/configuration-service.ts

```diff
@@ -16,7 +16,7 @@ export interface IConfigurationService {
 }
 
 export class ConfigurationService {
-	private static instance: ConfigurationService;
+	static instance: ConfigurationService;
```

Comment:
please try to find a way to write these test without the need for making this public

Reply:
I made it private again and just added a mock for the getInstance() method.